scripts/security/check_secret_expiration.py

```
@@ -0,0 +1,584 @@
+            f"**Contact:** security@toolboxai.com",
+            "",
+        ]
+    )
+
+    return "\n".join(report_lines)
+
+
+def generate_json_report(check_type: str, days: int = 7) -> Dict:
+    """Generate JSON report."""
+    report = {
+        "generated_at": datetime.now().isoformat(),
```

Ruff (0.14.4)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)